packages/react/package.json

```diff
@@ -14,21 +14,19 @@
     "lib/*"
   ],
   "scripts": {
-    "babel": "babel --ignore test.js,Tests.js,test -d lib src",
```

Comment:
Removing. This was my very quick first pass at attempting to support tree-shaking.